.github/workflows/cypress-test.yml

```diff
@@ -50,7 +50,7 @@ jobs:
           opensearch-version: ${{ env.OPENSEARCH_VERSION }}
           plugin-name: ${{ env.PLUGIN_NAME }}
           setup-script-name: setup
-          admin-password: admin
+          admin-password: myStrongPassword123!
```

Comment:
I may be out of date on this one, but I just wanna check are we still using this approach to configure the admin password? I remember we were trying to passing the flag in demo script for the admin password setup.

Reply:
This is a pretty tricky question here - we recently merged in changes into FTR repo, which will spin up 3.0.0 and 2.x > 2.11 with admin:myStrongPassword123!. So thus, we need to adapt this change here, as well - or else the FTR will not have the correct creds to access the OS backend. Unless we want to branch the FTR off from having that default password. However, that is currently needed in order to run integ test against the distribution in distribution builds.